libs/utils/utils.js

```diff
@@ -404,6 +404,9 @@ export const loadScript = (url, type) => new Promise((resolve, reject) => {
     if (type) {
       script.setAttribute('type', type);
     }
+    if (mode) {
+      script.setAttribute(mode, true);
+    }
```

Comment:
Nit: This can be a 1 liner
```suggestion
    if (mode) script.setAttribute(mode, true);
```